resources/electron/electron-plugin/dist/server/api/notification.js

```diff
@@ -1,26 +1,44 @@
 import express from 'express';
 import { Notification } from 'electron';
 import { notifyLaravel } from "../utils.js";
+import playSoundLib from 'play-sound';
+import fs from 'fs';
+const isLocalFile = (sound) => {
+    if (typeof sound !== 'string')
+        return false;
+    if (/^https?:\/\//i.test(sound))
+        return false;
+    return sound.includes('/') || sound.includes('\\');
+};
 const router = express.Router();
 router.post('/', (req, res) => {
     const { title, body, subtitle, silent, icon, hasReply, timeoutType, replyPlaceholder, sound, urgency, actions, closeButtonText, toastXml, event: customEvent, reference, } = req.body;
     const eventName = customEvent !== null && customEvent !== void 0 ? customEvent : '\\Native\\Desktop\\Events\\Notifications\\NotificationClicked';
     const notificationReference = reference !== null && reference !== void 0 ? reference : (Date.now() + '.' + Math.random().toString(36).slice(2, 9));
+    const usingLocalFile = isLocalFile(sound);
     const notification = new Notification({
         title,
         body,
         subtitle,
-        silent,
+        silent: usingLocalFile ? true : silent,
         icon,
         hasReply,
         timeoutType,
         replyPlaceholder,
-        sound,
+        sound: usingLocalFile ? undefined : sound,
         urgency,
         actions,
         closeButtonText,
         toastXml
     });
+    if (usingLocalFile && !silent) {
+        fs.access(sound, fs.constants.F_OK, (err) => {
+            if (err) {
+                return;
```

Comment:
It might be helpful to raise this error in the browser console.

I think you can do something like this:

``` js
import { broadcastToWindows } from '../utils.js';

broadcastToWindows('log', {level, message, context});
```